Code of Conduct updated grammar and punctuation (#2324)
privacyguides/privacyguides.org: docs/CODE_OF_CONDUCT.md

**We do not allow** inappropriate or otherwise unacceptable behavior, such as sexualized language, trolling and insulting comments, or otherwise promoting intolerance or harassment.

> ### Community Standards

Reviewer: If we want all the headers to be on the same level they should all be h2 tags instead of skipping h2 and going to h3.

```suggestion
## Community Standards
```

What we expect from members of our communities:

1. **Do not spread misinformation**  

      We are creating an evidence-based educational community around information privacy and security, not an information home for conspiracy theories. For example, when making a claim that a certain piece of software is malicious or that certain telemetry data is privacy invasive; explain in detail what is collected and how it collected. Claims of this nature must be backed by technical evidence.

2. **Do not abuse our willingness to help**  

      Our community members are not free tech support. We are happy to help with specific steps for individual's, privacy journey, if they are willing to put in effort. We are not obligated to answer endless, repetitive questions, about general computer problems solvable with a simple internet search. **Do not** become a [help vampire](https://slash7.com/2006/12/22/vampires/).
  
3. **Behave in a positive and constructive manner**

> Examples of behavior that contributes to a positive environment for our community include:

Reviewer: ```suggestion
      Examples of behavior that contributes to a positive environment for our community include:
```

This keeps the indentation correct under list item 3.

> - Being respectful of differing opinions, viewpoints, and experiences.
> - Demonstrating empathy and kindness toward others.
> - Focusing on what is best not just for us as overseers, but for the overall community.
> - Giving and gracefully accepting constructive feedback within' our community while growing and improving.
> - Operating with a communal mindset at all times.

Reviewer: ```suggestion
      - Being respectful of differing opinions, viewpoints, and experiences.
      - Demonstrating empathy and kindness toward others.
      - Focusing on what is best not just for us as overseers, but for the overall community.
      - Giving and gracefully accepting constructive feedback within' our community while growing and improving.
      - Operating with a communal mindset at all times.
```

> ### Unacceptable Behavior

Reviewer: ```suggestion
## Unacceptable Behavior
```

The following behaviors are considered harassment and are unacceptable within our community:

> - Any other conduct which would reasonably be considered inappropriate in a professional setting.

Reviewer: Not sure why it makes sense to re-order this list and make this the first item?

Author: I had reordered them for the purpose of flow-readability. It is one of the writing styles that I have (and I had some second thoughts about it) within' my writing style. 

I was curious what the overall opinion would be, as it is not necessary, but if it flows within' the word structure it is an option. 

Thank you for the question @jonaharagon !

- Public and/or private harassment of any kind.
- Publishing others' private information, such as a physical address and/or email address, without a persons explicit permission.
- The use of sexualized language or imagery, and sexual attention or advances of any kind.
- Trolling, insulting and/or derogatory comments, including personal or political attacks.

> ### Scope

Reviewer: ```suggestion
## Scope
```

> Our Code of Conduct applies within' all project spaces and when an individual is representing, the Privacy Guides project in other communities.

Reviewer: The current wording makes more grammatical sense.
```suggestion
Our Code of Conduct applies within all project spaces, as well as when an individual is representing the Privacy Guides project in other communities.
```

We are responsible for clarifying the standards of our community and have the right to remove or alter the comments of those participating within our community, as necessary and at our discretion.

> ### Contact

Reviewer: ```suggestion
## Contact
```

> If you observe a problem on a platform like Matrix or Reddit, please contact our moderators on that platform via chat, DM (Direct Messaging), or through any designated "Modmail" system.

Reviewer: ```suggestion
If you observe a problem on a platform like Matrix or Reddit, please contact our moderators on that platform via chat, direct message, or any designated "Modmail" system.
```

Author: For me it seems more streamlined; more ease of movement and (comprehension) readability.

Author: I should mention that further input with the word structure was for ease of use; readability for varying audience ages (young or old). Those in the community re-reading or who may be new; whether out loud or silent. It sounds overly complex, however I have good faith that the changes will benefit the project. 

Huge thanks to the community!
